reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Variables.md

```diff
@@ -105,14 +105,37 @@ $Splat = @{ Name = "Win*"; Include = "WinRM" }
 Invoke-Command -Session $s -ScriptBlock { Get-Service @Using:Splat }
 ```
 
-## Using local variables in PowerShell 2.0
+### Other situations where the 'Using' scope modifier is needed
+
+For any script or command that executes out of session, you need the `Using`
+scope modifier to embed variable values from the calling session scope, so that
+out of session code can access them. The `Using` scope modifier is supported in
+the following contexts:
+
+- Remotely executed commands, started with `Invoke-Command` using the
+  **ComputerName** or **Session** parameter (remote session)
+- Background jobs, started with `Start-Job` (out-of-process session)
+- Thread jobs, started via `Start-ThreadJob` (separate thread session)
+
+Depending on the context, embedded variables are copied values of the variables
+in the caller's scope. Out-of-process sessions, such as remote sessions,
```

Comment:
"Depending on the context, embedded variable values are either independent copies of the data in the caller's scope or references to it."

Reply:
"In remote and out-of-process sessions, they are always independent copies. In thread sessions, they are references to the caller's data if the data is an instance of a .NET reference type (as opposed to a value type)."

Reply:
I think we can omit the distinction between .Net reference and value types here and just say thread session data is passed by reference.

Reply:
Fixed